index.html

```diff
@@ -2,77 +2,114 @@
 <html>
     <head>
         <meta charset="UTF-8">
-
         <title>Uniter PHP NavMenuComponent demo</title>
-
         <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css">
+        <script src="/assets/jquery-full.js"></script>
```

Comment:
jQuery should be embedded in the compiled `bundle.js`, no need to include it separately here too